libsql-server/src/query_analysis.rs

```diff
@@ -180,7 +181,6 @@ impl StmtKind {
             | "read_uncommitted"
             | "recursive_triggers"
             | "reverse_unordered_selects"
-            | "schema_version"
```

Comment:
shouldn't this line be kept? isn't schema version with body still a write?

Reply:
schema version with body wasn't accepted before